apps/classes/management/commands/populate_default.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+
+from django.core.management.base import BaseCommand
+
+from apps.classes.models import *
+from apps.users.models import *
+
+
+class Command(BaseCommand):
+    help = "Populates 5 BasicClassInfos, 5 BasicClassSessions, 5 Instructors, 5 Users"
+
+    def handle(self, *args, **kwargs):
+
+        # Populating UserInfo.
+        ui_list = ["user1", "user2", "user3", "user4", "user5"]
+
+        for user in ui_list:
+            User.objects.create_user(user, user + "@gmail.com", user + "pw")
+
+        # Populating Instructors.
+        instructors_list = {
+            "Sanjay Madhav": "",
+            "David Matthias Kempe": "",
+            "Margaret Moser": "",
+            "Reuven Firestone": "",
+            "Aaron Cote": "",
+        }
+
+        for key, value in instructors_list:
+            value = Instructor.objects.create(name=key)
```

Comment:
i realized can't update dict like that lmfao
`instructors_list[key] = ...`

And do `update_or_create` <- so we don't create duplicate objects when we run this multiple times

Reply:
Doc: https://docs.djangoproject.com/en/4.0/ref/models/querysets/#update-or-create
It returns a tuple so look at what we need to save